docs/drafts/provided-serviceaccount.md

```diff
@@ -0,0 +1,34 @@
+# Provided ServiceAccount for ClusterExtension Installation and Management
+
+While OLMv0 provides many mechanisms for permission configuration via OperatorGroups,
+or by manipulating CatalogSources, cluster administrators must be aware of these options
+and actually implement them. If no ServiceAccount is explicitly specified for installing
+and upgrading operators, then cluster-admin is used by default. This can pose security risks
+by providing more permissions than are actually required for the management of any specific bundle.
```

Comment:
Not sure we really need to mention OLMv0 here - what do others think?

Reply:
I definitely see an argument for certain OLMv0 callouts on specific notable changes. And this is certainly one of those.

But maybe we should just have a separate doc that highlights all of those things in one place?

Reply:
That sounds like a good idea. So do you think I should remove that mention here and open a ticket to create a different page outlining the differences between OLMv0/v1?

Reply:
I'm +1 for a separate "changes from OLMv0" type documentation item

Reply:
I agree with this, we can have a separate document for changes from OLMv0 and not mention OLMv0 in this doc.

Reply:
Maybe we can rephrase as below:
```
OLMv1 differs in terms of permission management from its predecessor, it requires you to specify the permissions required to install cluster extensions upfront. It does not provider cluster admin privileges by default. Cluster administrators can specify the exact permissions that are actually required for the management of any specific bundle.

Reply:
I added the following to my doc:

OLMv1 does not provide cluster admin privileges by default for installing cluster extensions. It depends on the cluster extension developer to specify the permissions required for the management of any specific bundle. A ServiceAccount needs to be explicitly specified for installing and upgrading operators else will face errors when deploying your cluster extension.